unicoremc/views.py

```diff
@@ -23,7 +24,9 @@
 from unicoremc import tasks, utils
 
 
-def get_all_repos(request):
+def get_repos(refresh):
+    if not refresh:
+        return cache.get('repos')
```

Comment:
Happy for it to return None. It just means the user will have to do a manual refresh. 
I'm using the Travis model where you have to do a "Sync Repos" to get the list of latest repos.